Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
```

Comment:
Include `Pipfile` in `.gitignore` if you're setting repo up to use tox like telliot_core / telliot_feeds repos

Reply:
You gotta remove Pipfile still